doc/api/errors.md

```diff
@@ -2021,13 +2021,6 @@ An invalid `options.protocol` was passed to `http.request()`.
 Both `breakEvalOnSigint` and `eval` options were set in the [`REPL`][] config,
 which is not supported.
 
-<a id="ERR_INVALID_REPL_INPUT"></a>
-
-### `ERR_INVALID_REPL_INPUT`
-
-The input may not be used in the [`REPL`][]. The conditions under which this
-error is used are described in the [`REPL`][] documentation.
-
```

Comment:
We should move this to the ad-hoc section https://github.com/nodejs/node/blob/cdae31570669cd24b1251f2be0243230c53344cf/doc/api/errors.md#L3257-L3260